aten/src/ATen/native/transformers/cuda/flash_attn/flash_api.cpp

```diff
@@ -389,20 +389,17 @@ mha_fwd(const at::Tensor &q,         // batch_size x seqlen_q x num_heads x head
         std::optional<at::Generator> gen_) {
 
     auto dprops = at::cuda::getCurrentDeviceProperties();
-    // bool is_sm75 = dprops->major == 7 && dprops->minor == 5;
-    bool is_sm8x = dprops->major == 8 && dprops->minor >= 0;
-    bool is_sm90 = dprops->major == 9 && dprops->minor == 0;
-    bool is_sm10x = dprops->major == 10 && dprops->minor >= 0;
-    bool is_sm120_or_sm121 = dprops->major == 12 && dprops->minor <= 1;
-    TORCH_CHECK(is_sm120_or_sm121 || is_sm10x || is_sm90 || is_sm8x, "FlashAttention only supports Ampere GPUs or newer.");
+    // bool is_sm75_or_newer = (dprops->major * 10 + dprops->minor) >= 80;
+    bool is_sm80_or_newer = (dprops->major * 10 + dprops->minor) >= 80;
+    TORCH_CHECK(is_sm80_or_newer, "FlashAttention only supports Ampere GPUs or newer.");
     // We will support Turing in the near future
-    // TORCH_CHECK(is_sm90 || is_sm8x || is_sm75, "FlashAttention only supports Turing GPUs or newer.");
```

Comment:
This comment also seems stale, I think at the time there was plans to support turing but never came about, lets just delete

Reply:
same for all the the ones below lol